internal/operands/vm-console-proxy/reconcile.go

```diff
@@ -252,30 +252,28 @@ func reconcileRoute(serviceName string) common.ReconcileFunc {
 }
 
 func isEnabled(request *common.Request) bool {
-	if request.Instance.GetAnnotations() == nil {
-		return false
-	}
-	enabledStr, ok := request.Instance.GetAnnotations()[EnableAnnotation]
-	if !ok {
-		return false
-	}
-	isEnabled, err := strconv.ParseBool(enabledStr)
-	if err != nil {
-		return false
+	if request.Instance.GetAnnotations() != nil {
+		enable, isFound := request.Instance.GetAnnotations()[EnableAnnotation]
+		if isFound {
+			isEnabled, err := strconv.ParseBool(enable)
+			if err == nil {
+				return isEnabled
+			}
+		}
 	}
-	return isEnabled
+
+	return false
 }
 
 func getVmConsoleProxyNamespace(request *common.Request) string {
-	const defaultNamespace = "kubevirt"
-	if request.Instance.GetAnnotations() == nil {
-		return defaultNamespace
-	}
-	namespaceStr, ok := request.Instance.GetAnnotations()[VmConsoleProxyNamespaceAnnotation]
-	if !ok {
-		return defaultNamespace
+	if request.Instance.GetAnnotations() != nil {
+		namespace, isFound := request.Instance.GetAnnotations()[VmConsoleProxyNamespaceAnnotation]
```

Comment:
You could merge L270 and 271.

Reply:
Do you mean:
```
if namespace, isFound := request.Instance.GetAnnotations()[VmConsoleProxyNamespaceAnnotation]; isFound {
			return namespace
		}
```
If yes, is it common code style? Because I don't like it and feel it's line too long for good code readability.

Reply:
It is common, but It's up to you to decide. We don't have guidelines about this.